add-on/panels/test/panel_test.js

```diff
@@ -944,22 +944,6 @@ describe("loop.panel", function() {
       sinon.assert.calledOnce(fakeWindow.close);
     });
 
-    it("should render the no rooms view when no rooms available", function() {
-      var view = createTestComponent();
-      var node = view.getDOMNode();
-
-      expect(node.querySelectorAll(".room-list-empty").length).to.eql(1);
-    });
```

Comment:
We should still check for what should be handled when there's no rooms available.